configure.ac

```diff
@@ -193,6 +193,7 @@ AS_IF([test "x$with_asio"   = "xno"], [systems=`echo $systems|tr ' ' \\\\n|grep
 AS_IF([test "x$with_dsound" = "xno"], [systems=`echo $systems|tr ' ' \\\\n|grep -v dsound`])
 AS_IF([test "x$with_wasapi" = "xno"], [systems=`echo $systems|tr ' ' \\\\n|grep -v wasapi`])
 systems=" `echo $systems|tr \\\\n ' '` "
+need_pthread=yes
```

Comment:
I'm not a Windows user. But I think especially use cases without mingw and msys2 should be tested for regressions. Were WASAPI et al compiled without pthread before? Are they still working? Should this need_pthread be put in an if clause for Pulseaudio and JACK?

However RtAudio development is slow and there should be plenty of time to fix problems that occur with this change.

Just saw that you also cleaned up the wchar conversion. Nice.